controllers/profiles/common/object_creators.go

```diff
@@ -202,6 +216,85 @@ func ServiceCreator(workflow *operatorapi.SonataFlow) (client.Object, error) {
 	return service, nil
 }
 
+// SinkBindingCreator is an ObjectsCreator for SinkBinding.
+// It will create v1.SinkBinding based on events defined in workflow.
+func SinkBindingCreator(workflow *operatorapi.SonataFlow) (client.Object, error) {
+	lbl := workflowproj.GetDefaultLabels(workflow)
+
+	// skip if no produced event is found
+	if workflow.Spec.Sink == nil || !workflowdef.ContainsEventKind(workflow, cncfmodel.EventKindProduced) {
```

Comment:
The cases we chatted, regarding the "timeouts" and the situations where a workflow might produce no regular events, but can still send cloud events to the jobs service. Are you going to implement in a complementary PR?

Reply:
yes, I will do it in PR for job services: https://github.com/apache/incubator-kie-kogito-serverless-operator/issues/368. this is gonna be my next pr

Reply:
btw, I'm implementing this calculation already in my upcoming PR. I'll let you know when merged.

Reply:
great! Thank you